nrt/data/SMIB/SMIB_BasicTestCases/SMIB_1_StepPm_IIDM/reference/outputs/initValues/globalInit/dumpInitValues-line1.txt

```diff
@@ -26,5 +26,9 @@ line_YPu_im                                        =       0.000000
 line_YPu_re                                        =       0.000000
 line_ZPu_im                                        =       0.022522
 line_ZPu_re                                        =       0.000000
+line_Running0                                      =       1.000000
+line_SwitchOffSignal10                             =      -1.000000
+line_SwitchOffSignal20                             =      -1.000000
+line_SwitchOffSignal30                             =      -1.000000
```

Comment:
Here for example SwitchOffSignal30 can be set to true and this will change the value of Running0 -> false. But since line does not have 3 switch-off signals, it does not seem to affect the value of running that stays = true.
Which is good.